epi_scanner/app.py

```diff
@@ -46,10 +46,10 @@
 warnings.filterwarnings("ignore")
 
 DATA_TABLE = None
-DUCKDB_FILE = Path(os.path.join(
-    str(EPISCANNER_DUCKDB_DIR), "episcanner.duckdb")
-)
-
+#DUCKDB_FILE = Path(os.path.join(
+#    str(EPISCANNER_DUCKDB_DIR), "episcanner.duckdb")
+#)
+DUCKDB_FILE = Path('/Users/eduardoaraujo/episcanner/episcanner.duckdb')
```

Comment:
Careful here with leaving local paths in the code. This will break the code in production.
Revert the code to what it was and to make it work in your dev environment, create the `EPISCANNER_DUCKDB_DIR` variable in your local .env file.